### DISCUSSION TITLE
Add support for sending images by pasting from the clipboard

### DISCUSSION THREAD

**Discussion by Flaviongas** [d99kris/nchat#185] [Ideas]
I usually send screenshots in Whatsapp, but to send a screenshot i have to convert it to a file and then send it.

**Comment by d99kris** [comment]
Hi @Flaviongas - thanks for reporting a feature request. Given this is a bit of a corner-case and there are various complexities to handle (especially if handling pasting files in general), I'm not sure I will get to this any time soon. But I will leave the request here..

**Comment by d99kris** [comment]
Support for sending images by pasting from the clipboard has been implemented in e673d06 / `v5.11.26`. To enable the functionality install `libpng-dev` or equivalent, and build latest nchat. Look out for the following to confirm libpng headers are found:
```
-- Looking for png.h - found
-- Clipboard: Text, Image
```

Use alt/opt-v to paste images. By default nchat prompts a confirmation dialog, which may be disabled by setting `confirm_send_pasted_image=0` in `ui.conf`
